x-pack/plugins/security_solution/public/timelines/components/field_renderers/field_renderers.tsx

```diff
@@ -192,212 +167,3 @@ export const whoisRenderer = (ip: string) => <WhoIsLink domain={ip}>{i18n.VIEW_W
 export const reputationRenderer = (ip: string): React.ReactElement => (
   <ReputationLink domain={ip} direction="column" />
 );
-
```

Comment:
`DefaultFieldRendererComponent`, `MoreContainer` and `DefaultFieldRendererOverflow` moved to different files.

Reply:
Remember to update the path of the components that consumed `DefaultFieldRenderer`